src/main/java/org/fungover/storm/httphandler/Request.java

```diff
@@ -0,0 +1,74 @@
+package org.fungover.storm.httphandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Request {
+    private byte[] body;
+    private String url;
+    private final Map<String, String> parameters;
+    private final Map<String, String> headers;
+
+    public Request() {
+        this.headers = new HashMap<String, String>();
```

Comment:
```suggestion
        this.headers = new HashMap<>();
```

Reply:
Good suggestion! I'm not sure why String,String is definied there since it's unnecessary.